sqlalchemy-stubs/sql/schema.pyi

```diff
@@ -193,9 +194,9 @@ class Column(DialectKWArgs, SchemaItem, ColumnClause[_TE]):
         onupdate: Optional[Any] = ...,
         primary_key: bool = ...,
         server_default: Optional[
-            Union[FetchedValue, str, TextClause, ColumnElement[Any]]
+            Union[FetchedValue, FunctionElement, str, TextClause, ColumnElement[Any]]
```

Comment:
I think we should create an alias for this.

Reply:
@CaselIT any suggestion off hand for the alias?

As in, are you thinking maybe an alias for this particular lengthy `Union` in full or for some subset of it that might occur most often?

Tomorrow, I'll check into finding the cases I missed, look into what might make a reasonable alias, and take a look into what you all are doing for tests. Writing tests for stubs is next level meta, I'll have to take a look at your project's existing tests to see what your patterns look like!

Reply:
> @CaselIT any suggestion off hand for the alias?
> 
> As in, are you thinking maybe an alias for this particular lengthy `Union` in full or for some subset of it that might occur most often?
> 
> Tomorrow, I'll check into finding the cases I missed, look into what might make a reasonable alias, and take a look into what you all are doing for tests. Writing tests for stubs is next level meta, I'll have to take a look at your project's existing tests to see what your patterns look like!

Just an alias for the lengthy union. Something like
`_ServerDefaulyType = + Union[FetchedValue, FunctionElement, str, TextClause, ColumnElement[Any]]`

Regarding the tests, they are just files that are type checked with mypy. They don't have to be runnable with python

Reply:
Hi @CaselIT, when you get a chance can you check into the newer revision I pushed?

I added aliases and tests.

I'm erring on the side of caution/simplicity by using `FunctionElement` in the `server_onupdate` type alias `Union` instead of `ColumnElement[...]` (a much broader type) as I am not sure if `server_onupdate` support is exactly in line with `server_default` in that regard but the documentation (and first hand testing) show at least `FunctionElement` is permitted so my test cases and second type alias focus on that.

If the two keyword arguments have identical feature support, the types could be merged later. I just didn't want to make that assumption.

Reply:
makes sense to me. Thanks!